### PR TITLE
iptables: implement log_level parameter

### LINKED ISSUE #25100: Support logging in the 'iptables' module
##### ISSUE TYPE
 - Feature Idea

##### COMPONENT NAME
iptables

##### ANSIBLE VERSION
```
ansible 2.3.0.0 (stable-2.3 4b5553c97f) last updated 2017/04/23 13:15:41 (GMT +200)
  config file =
  configured module search path = Default w/o overrides
  python version = 2.7.9 (default, Mar  1 2015, 12:57:24) [GCC 4.9.2]
```

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
N/A

##### SUMMARY
I propose to support logging in the iptables module with two additional parameters that map directly to iptables (in both versions ipv4 and ipv6).

* log_prefix
* log_level

For example the following iptables command
```
iptables -A LOGGING -m limit --limit 2/second --limit-burst 20 -j LOG --log-prefix 'iptables: ' --log-level info
```
becomes
```
iptables:
  chain: LOGGING
  action: append
  state: present
  limit: 2/second
  limit_burst: 20
  log_prefix: "iptables: "
  log_level: info
```

##### STEPS TO REPRODUCE
N/A

##### EXPECTED RESULTS
N/A

##### ACTUAL RESULTS
N/A

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Fixes: #25100

Based upon https://github.com/ansible/ansible/pull/25118

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
lib/ansible/modules/system/iptables.py

### CODE REVIEW

**Comment by Akasurde** [comment]
@JoelSanchez @LinusU Could you please review this and let me know if it works ? Thanks.

**Comment by LinusU** [comment]
I don't have access to a Linux computer to test this at the moment, but the code looks good 👍

**Comment by jillr** [comment]
Hi @Akasurde, could you add unit test coverage for this param please? I can give this a test next week when I'm back in the office.

File: lib/ansible/modules/system/iptables.py [hunk 1]
```diff
@@ -150,6 +150,14 @@
       - Specifies a log text for the rule. Only make sense with a LOG jump.
     type: str
     version_added: "2.5"
+  log_level:
+    description:
+      - 'Logging level according to the syslogd-defined priorities.'
```
Comment (goneri) [inline lib/ansible/modules/system/iptables.py]: No need to quote the string here.

**Review by goneri — APPROVED** [review]

**Comment by goneri** [comment]
shipit

**Pull request merged**